core/spec.md

```diff
@@ -436,6 +445,114 @@ attributes. However they MUST adhere to the following rules:
   potential conflicts with future Registry specification attributes. For
   example, use of a model (or domain) specific prefix could be used.
 
+#### `xid` - Referencing xRegistry Entities
+
+An `xid` is a URI according to
+[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) that
+uniquely identifies an entity within an xRegistry. Its purpose is to
+reference the entity and express a relation to it, even if it is not stored in
+the local registry. The `xid` can also be used in other places where it
+is helpful to use a URI, e.g. events or knowledge graphs.
+Each entity has exactly one `xid`. Therefore, comparing two `xid`
+values is possible to determine, if they identify the same or different
+entities. To achieve this, an xRegistry-specific URI scheme `xreg` is defined:
+
+`xid` = `"xreg://" authority "/" entity-path "/" entity-id`
+
+`entity-path` = `group-path` / `resource-path` / `version-path`
+
+`group-path`= `GROUPs "/" gID`
+
+`resource-path`= `group-path "/" RESOURCEs "/" rID`
+
+`version-path`= `resource-path "/" versions "/" vID`
+
+Where:
+
+- `GROUPs` is a Group type name (plural). e.g. `endpoints`.
+- `GROUP`, not shown, is the singular name of a Group type.
+- `gID` is the `id` of a single Group.
+- `RESOURCEs` is a Resource type name (plural). e.g. `definitions`.
+- `RESOURCE`, not shown, is the singular name of a Resource type.
+- `rID` is the `id` of a single Resource.
+- `vID` is the `id` of a single Version of a Resource.
+
+##### `authority`
+
+The `authority` is a string that uniquely identifies the entity's authority.
```

Comment:
Valid chars?

Reply:
Good question. If someone decides to use a valid domain name, it is restricted by the domain name system, but that is not necessarily a constraint for all `authorities` in xRegistry. I suppose it would make sense to apply the same rules we have for RESOURCE/GROUP/versionid. WDYT? @duglin